src/ast/variables/ExportDefaultVariable.ts

```diff
@@ -14,6 +14,7 @@ export default class ExportDefaultVariable extends LocalVariable {
 		super(name, exportDefaultDeclaration, exportDefaultDeclaration.declaration);
 		this.isDefault = true;
 		this.hasId = !!(<FunctionDeclaration | ClassDeclaration>exportDefaultDeclaration.declaration).id;
+		this.exportName = 'default';
```

Comment:
How hard would it be to include the filename of the module as part of this name, e.g., `my_module$default` for the default export of `my-module.js`?  Seems like this could make for easier debugging in some cases.

Reply:
The reason we use the initial name is because this stage of the algorithm is still attempting to determine if the chunk is an entry point module itself (entry.js), or if it is a chunk (chunk[i].js). Ideally the algorithm could do the export naming after this stage in the process in which case we can better handle naming definitely. I will look into a refactor along these lines, but perhaps not just yet.

Reply:
(So basically in the scenario that the chunk is an entry point, we need it to do the correct naming as the entry point would expect)